<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtPhysicsEngineClass</class>
 <widget class="QMainWindow" name="QtPhysicsEngineClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtPhysicsEngine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QtGLWidget" name="openGLWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoadXML"/>
    <addaction name="actionSaveXML"/>
    <addaction name="actionLoad_Obj"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionLoadXML">
   <property name="text">
    <string>Load XML...</string>
   </property>
  </action>
  <action name="actionSaveXML">
   <property name="text">
    <string>Save XML...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLoad_Obj">
   <property name="text">
    <string>Load Obj...</string>
   </property>
  </action>
  <action name="actionSave_Obj">
   <property name="text">
    <string>Save Obj...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtGLWidget</class>
   <extends>QWidget</extends>
   <header>qtglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PhysicsEngineLib.qrc"/>
 </resources>
 <connections/>
</ui>
